scripts/pi-setup/install-pi-dependencies.sh

```diff
@@ -16,24 +16,35 @@
  # limitations under the License.
 set -e
 
-# Silence user prompts about reboot and service restart required (script will prompt user to reboot in the end)
+ROOT_DIR=$(realpath $(dirname "$0")/../..)
+SCRIPT_DIR="$ROOT_DIR/scripts"
+UBUNTU_SCRIPT_DIR="$SCRIPT_DIR/ubuntu"
```

Comment:
This doesn't seem to be used in this file. Please consider removing this variable.